_includes/docs/latest/required_roles.md

```diff
@@ -1,30 +1,23 @@
-**Forseti server service account requires:
  
 ***Granted at the Organization level**
   
- * `roles/appengine.appViewer`
- * `roles/bigquery.dataViewer`
- * `roles/browser`
- * `roles/cloudsql.viewer`
- * `roles/compute.networkViewer`
- * `roles/compute.securityAdmin`
- * `roles/iam.securityReviewer`
- * `roles/servicemanagement.quotaViewer`
- * `roles/serviceusage.serviceUsageConsumer`
+ * `roles/appengine.appViewer` (server)
+ * `roles/bigquery.dataViewer` (server)
+ * `roles/browser` (server)
+ * `roles/cloudsql.viewer` (server)
+ * `roles/compute.networkViewer` (server)
+ * `roles/compute.securityAdmin` (server)
+ * `roles/iam.securityReviewer` (server)
+ * `roles/servicemanagement.quotaViewer` (server)
+ * `roles/serviceusage.serviceUsageConsumer` (server)
  
 ***Granted at the project level**
  
- * `roles/cloudsql.client`
- * `roles/logging.logWriter`
- * `roles/storage.objectViewer`
- * `roles/storage.objectCreator`
+ * `roles/cloudsql.client` (server)
+ * `roles/logging.logWriter` (server, client)
+ * `roles/storage.objectViewer` (server, client)
```

Comment:
Why don't you use these includes instead?
- https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-forsetisecurity.org-dev/_includes/docs/latest/forseti-server-gcp-required-roles.md
- https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-forsetisecurity.org-dev/_includes/docs/latest/forseti-client-gcp-required-roles.md

This this include can go away.

Reply:
done